contracts/common/tracer/ContractCaller.sol

```diff
@@ -0,0 +1,156 @@
+pragma solidity >=0.8.0;
+
+import "./ContractCallee.sol";
+
+contract ChildContract {
+    address public childAddr;
+
+    constructor() payable {
+        childAddr = address(this);
+    }
+}
+
+contract ContractCaller {
+    uint256 public parameter;
+    address public sender;
+
+    event EventContractCaller(string text);
+    function deposit() public payable {}
+
+    function setParamWithDelegateCall(
+        address _contractCallee,
+        uint256 _param
+    ) public {
+        (bool success, ) = _contractCallee.delegatecall(
+            abi.encodeWithSignature("setParam(uint256)", _param)
+        );
+        require(success);
+    }
+
+    function setParamWithCallcode(
```

Comment:
I can't find where you use this function

Reply:
deleted